azure-pipelines.yml

```diff
@@ -153,6 +153,7 @@ stages:
 - ${{ if and(ne(variables['System.TeamProject'], 'public'), notin(variables['Build.Reason'], 'PullRequest')) }}:
   - template: eng\common\templates\post-build\post-build.yml
     parameters:
+      publishingInfraVersion: 3
```

Comment:
Just noticed the issue mentioned in the line below was resolved 2 years ago. @JohnTortugo 
https://github.com/dotnet/arcade/issues/2871